### PR TITLE
Cleaned up adfs files

### PR DESCRIPTION
**Pull request by tamigoodall**
Removed excessive MS screenshots
Cleaned up steps, headings, and grammar
Added two troubleshooting cases and fixes

https://docs-content-staging-pr-8037.herokuapp.com/docs/connections/enterprise/adfs
https://docs-content-staging-pr-8037.herokuapp.com/docs/protocols/saml/adfs
https://docs-content-staging-pr-8037.herokuapp.com/docs/protocols/saml/saml-configuration/troubleshoot/auth0-as-sp
https://docs-content-staging-pr-8037.herokuapp.com/docs/protocols/saml/saml-configuration/troubleshoot/auth0-as-idp

### CODE REVIEW

File: articles/connections/enterprise/adfs.md [hunk 1]
```diff
@@ -22,45 +22,41 @@ useCase:
...
 ---
-# Connect your app to ADFS
+# Connect Your Application to Microsoft ADFS
 
-First, provide this information to your ADFS administrator:
+To connect your application to Microsoft ADFS, you will need to provide the following information to your Active Directory Federation Services (ADFS) administrator:
 
 * Realm Identifier: `urn:auth0:${account.tenant}`
-* Endpoint: `https://${account.namespace}/login/callback` (or, if you are using the [custom domains](/custom-domains) feature, `https://<YOUR CUSTOM DOMAIN>/login/callback`)
+* Endpoint: `https://${account.namespace}/login/callback` or `https://<YOUR CUSTOM DOMAIN>/login/callback`, if you are usinga  [custom domain](/custom-domains).
```
Comment (erjohnson) [inline articles/connections/enterprise/adfs.md]: using a*

File: articles/protocols/saml/saml-configuration/troubleshoot/common-saml-errors.md [hunk 2]
```diff
@@ -90,3 +90,11 @@ One common error is specifying the incorrect response protocol on the IdP-Initia
 2. Find your Connection, and click on **Settings**.
 3. Switch to the *IdP-Initiated* tab.
 4. Check the value you have set in the **Response Protocol** field.
+
+## Issue: User isn't logged out from the IdP
+
+When ADFS as configured as SAML IdP, if on the ADFS relaying party trust `Name ID` attribute isn't mapped the logout flow fails. For example, with the federated parameter `v2/logout?federated&...` user isn't redirected to the ADFS SAML logout endpoint but redirects back to application callback URL directly. As a consequence, the user isn't logged out from the IdP in that case.
```
Comment (erjohnson) [inline articles/protocols/saml/saml-configuration/troubleshoot/common-saml-errors.md]: - When ADFS is configured*
- if the ADFS relaying*

File: articles/connections/enterprise/adfs.md [hunk 3]
```diff
@@ -22,45 +22,41 @@ useCase:
     - customize-connections
     - add-idp
 ---
-# Connect your app to ADFS
+# Connect Your Application to Microsoft ADFS
 
-First, provide this information to your ADFS administrator:
+To connect your application to Microsoft ADFS, you will need to provide the following information to your Active Directory Federation Services (ADFS) administrator:
```
Comment (nicosabena) [inline articles/connections/enterprise/adfs.md]: As far as I can tell, MS calls the product "Active Directory Federation Services", without the "Microsoft" or "Windows" words in it. Our dashboard also says only "ADFS".
Maybe "To connect your application to Microsoft's ADFS"?

File: articles/protocols/saml/adfs.md [hunk 4]
```diff
@@ -4,202 +4,137 @@ toc: true
...
 useCase:
   - add-idp
+  - saml-adfs
 ---
 
-# ADFS SAML Connection
+# Setup a Microsoft ADFS SAML Connection
 
-Auth0 allows you to create a custom <dfn data-key="security-assertion-markup-language">SAML</dfn> connection to Microsoft's Active Directory Federation Services. In addition to getting a bit more flexibility when configuring your mappings, the SAML Connection allows you identity provider-initiated flows (this is something that you cannot do with WS-Fed).
+Create a custom <dfn data-key="security-assertion-markup-language">SAML</dfn> connection to Microsoft's Active Directory Federation Services (ADFS) to get more flexibility when configuring your mappings. The SAML Connection allows you identity provider-initiated flows which you cannot do with WS-Fed.
```
Comment (nicosabena) [inline articles/protocols/saml/adfs.md]: I would change "The SAML Connection allows you identity provider-initiated flows which you cannot do with WS-Fed." to "In addition, SAML connections allow [IdP-Initiated flows](protocols/saml/idp-initiated-sso), which are not supported with regular ADFS-type connections.". Or, considering that we don't recommend IdP-initiated flows, I'd just remove that sentence altogether.

File: articles/protocols/saml/adfs.md [hunk 5]
```diff
@@ -4,202 +4,137 @@ toc: true
...
+Optionally, you can sign your SAML requests to the ADFS server.
 
-1. Go to the **Settings** page for your SAMLP Identity Provider in the Dashboard
+1. Go to the **Settings** page for your SAMLP Identity Provider in the Auth0 Dashboard.
 2. Enable **Sign Requests**.
 3. Just below the **Sign Requests** toggle is a link to download your certificate.
 4. Return to ADFS and load the downloaded certificate using the **Signatures** tab of the Relying Party properties dialog.
 
-### Map Your Claims
+## Optional: Map additional claims
```
Comment (nicosabena) [inline articles/protocols/saml/adfs.md]: This step is mandatory, not optional. As part of the instructions, you can add a suggested mapping:

   | LDAP Attribute | Outgoing Claim |
    | - | - |
    | E-Mail-Addresses | E-Mail Address |
    | Display-Name | Name |
    | User-Principal-Name | Name ID |
    | Given-Name | Given Name |
    | Surname | Surname |

Also, make a note that the `NameID` outgoing claim should always be present to ensure correct session handling (later on there's a troubleshooting entry for logouts that mentions that this claim is required).

File: articles/protocols/saml/saml-configuration/troubleshoot/index.md [hunk 6]
```diff
@@ -51,12 +51,6 @@ When troubleshooting, we recommend beginning by gathering information that helps
 4. What is the expected behavior? What is the behavior you're seeing?
 5. How far through the login sequence does the user get?
 
-## Troubleshooting Approaches
-
-* [SAML Troubleshooting When Auth0 is the Service Provider](/protocols/saml/saml-configuration/troubleshoot/auth0-as-sp)
-
-* [SAML Troubleshooting When Auth0 is the Identity Provider](/protocols/saml/saml-configuration/troubleshoot/auth0-as-idp)
-
 ## If the Issue Affects Only One (or Just a Few) Users
```
Comment (nicosabena) [inline articles/protocols/saml/saml-configuration/troubleshoot/index.md:59]: I'm not quite sure about removing the "Troubleshooting approaches". Without them as the page content, this whole page has little substance, and the "Keep reading" list of articles is easily missed IMO.

File: articles/protocols/saml/adfs.md (hunk 4 above)
Comment (tamigoodall) [reply articles/protocols/saml/adfs.md]: Removed

File: articles/protocols/saml/adfs.md (hunk 5 above)
Comment (tamigoodall) [reply articles/protocols/saml/adfs.md]: Updated

File: articles/protocols/saml/saml-configuration/troubleshoot/index.md (hunk 6 above)
Comment (tamigoodall) [reply articles/protocols/saml/saml-configuration/troubleshoot/index.md:59]: Added back in

File: articles/protocols/saml/adfs.md [hunk 7]
```diff
@@ -4,202 +4,139 @@ toc: true
...
-Using the left-hand navigation pane, go to **ADFS > Service > Certificates**. Select the **Token-signing** certificate, and right click to select **View Certificate**.
-
-![](/media/articles/protocols/saml-adfs/saml15.png)
-
-On the **Details** tab, click **Copy to File...**.
-
-![](/media/articles/protocols/saml-adfs/saml16.png)
-
-This launches the Certificate Export Wizard. Click **Next** to proceed.
+1. To get your Federation Metadata, navigate to the following URL:
```
Comment (nicosabena) [inline articles/protocols/saml/adfs.md]: I believe that the whole "Obtain the federation metadata" step can be omitted. The sole purpose is to obtain the callback URL (ACS URL), which is always in the form of `https://{auth0_domain}/login/callback?connection={connection_name}`. This format is explained below, so no need to do this step.

File: articles/protocols/saml/adfs.md [hunk 8]
```diff
@@ -4,202 +4,139 @@ toc: true
...
 
-![](/media/articles/protocols/saml-adfs/saml24.png)
+2. Click **Save**. A page with instructions for creating a new Relying Party Trust in ADFS appears displaying the exact values required for your Auth0 account/connection. Make a note of these values. 
 
-Populate the **Trusted URL** with the **Post-back URL** value.
+    Here is an example: 
+    
+    | Parameter | Example Value |
+    | - | - |
+    | Post-back URL | `https://{yourAuth0accountdomain}/login/callback}/login/callback?connection={your new SAML connection}` |
```
Comment (nicosabena) [inline articles/protocols/saml/adfs.md]: I'd be good to explain either here or below in Trusted URL step of "Edit the Relying Party Trust" that {yourAuth0accountdomain} refers to the custom domain is one is being used.

File: articles/protocols/saml/adfs.md (hunk 7 above)
Comment (tamigoodall) [reply articles/protocols/saml/adfs.md]: Deleted

File: articles/protocols/saml/adfs.md (hunk 8 above)
Comment (tamigoodall) [reply articles/protocols/saml/adfs.md]: Added note

**Review by erjohnson — APPROVED** [review]

**Pull request merged**